### DISCUSSION TITLE
Automatic credit granting on SONA in jsPsych?

### DISCUSSION THREAD

**Discussion by BrianOdegaard** [jspsych/jsPsych#1099] [Ideas]
Hi everyone - 

I was wondering if anyone has successfully implemented automatic credit granting in jsPsych studies that are launched through URLs posted on SONA?  SONA Systems integrates with Qualtrics, Pavlovia, and other online platforms already.

This thread looked relevant:
https://github.com/jspsych/jsPsych/discussions/994

Was curious if anyone else out there has tried this and has a good method to use with jsPsych.
Thx!
Brian Odegaard

**Comment by jodeleeuw** [comment]
Hi Brian,

It's been a while since I've used SONA, but my memory from 5ish years ago is that SONA provides a completion URL to redirect participants to when the experiment is done. Is that still accurate?

**Comment by BrianOdegaard** [comment]
We're looking into modifying the study URL in the way they advise in the
examples for Qualtrics/PsychoPy/etc. on this page:
https://www.sona-systems.com/help/

I *think* it should work?  I was just curious if anyone out there had
tested it.

Right now we're following the method in Shashi Kant's Google Sheets
tutorial to generate our study URLs:
https://medium.com/@shashikg.iitk/jspsychsheet-running-online-behavioral-experiments-using-jspsych-and-google-sheet-adf29fb4d1a0
Not sure if you can actually modify those in the way that we're hoping to.
We'll see how testing this afternoon goes.

B

On Fri, Oct 2, 2020 at 1:59 PM Josh de Leeuw <notifications@github.com>
wrote:

> Hi Brian,
>
> It's been a while since I've used SONA, but my memory from 5ish years ago
> is that SONA provides a completion URL to redirect participants to when the
> experiment is done. Is that still accurate?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jspsych/jsPsych/discussions/1099#discussioncomment-89559>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACASZUCQGB5HQCR2QDH7HCDSIYIGXANCNFSM4SB7JCSQ>
> .
>


-- 
Brian Odegaard, PhD
Assistant Professor
Department of Psychology, University of Florida

This email may contain material that is confidential and/or privileged for
the sole use of the intended recipient. Any review, reliance or
distribution by others or forwarding without express permission is strictly
prohibited. If you are not the intended recipient, please contact the
sender and delete all copies.
